util/install/opensuse.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+_qmk_install_prepare() {
+    case $(grep ID /etc/os-release) in
+        *15.1*)
+            REPO_RELEASE=Leap_15.1;;
+        *15.2*)
+            REPO_RELEASE=Leap_15.2;;
+        *)
+            #REPO_RELEASE=Tumbleweed;;
+            echo "ERROR: Tumbleweed is currently not supported."
+            exit 1
+    esac
+
+    sudo zypper addrepo https://download.opensuse.org/repositories/devel:gcc/openSUSE_$REPO_RELEASE/devel:gcc.repo
+    sudo zypper addrepo https://download.opensuse.org/repositories/hardware/openSUSE_$REPO_RELEASE/hardware.repo
+    sudo zypper --gpg-auto-import-keys refresh
+}
+
+_qmk_install() {
+    echo "Installing dependencies"
+
+    sudo zypper install -y \
+        make clang gcc unzip wget zip \
+        python3-pip \
+        cross-avr-binutils cross-avr-gcc8 avr-libc \
+        cross-arm-binutils cross-arm-none-gcc8 cross-arm-none-newlib-devel \
+        avrdude dfu-programmer dfu-util
+
+    python3 -m pip install --user -r $QMK_FIRMWARE_DIR/requirements.txt
+}
```

Comment:
[There is no Tumbleweed directory for `devel:gcc`](https://download.opensuse.org/repositories/devel:/gcc/), and `cross-arm-none-gcc8` throws a bunch of linker errors on Leap... so OpenSUSE support is broken at this time. If any OpenSUSE user is able to provide a fix (or better yet, fix the packages and repos), it would be much appreciated!

Reply:
Would it be worth grabbing the official ARM toolchain in this circumstance, then?
https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads

Reply:
I'd kinda like to avoid another ~/qmk_utils folder if at all possible, considering you have to modify the .bashrc to add it to PATH (and some users may not be running bash), and then a new terminal must be opened. Perferably the already existing package should be fixed.